aichatweb-aspire.AppHost/Program.cs

```diff
@@ -0,0 +1,18 @@
+var builder = DistributedApplication.CreateBuilder(args);
+
+// You will need to set the endpoint and key to your own values
+// You can do this using Visual Studio's "Manage User Secrets" UI, or on the command line:
+//   cd this-project-directory
+//   dotnet user-secrets set Parameters:gitHubModelsToken YOUR-GITHUB-TOKEN
+var gitHubModelsToken = builder.AddParameter("gitHubModelsToken", secret: true);
+
+var ingestionCache = builder.AddSqlite("ingestionCache")
+    .WithSqliteWeb();
+
+var webApp = builder.AddProject<Projects.aichatweb_aspire_Web>("aichatweb-app");
+webApp.WithEnvironment("GITHUB_MODELS_TOKEN", gitHubModelsToken);
+webApp
+    .WithReference(ingestionCache)
+    .WaitFor(ingestionCache);
```

Comment:
Is there are a reason why these aren't just chained?

```suggestion
builder.AddProject<Projects.aichatweb_aspire_Web>("aichatweb-app")
    .WithEnvironment("GITHUB_MODELS_TOKEN", gitHubModelsToken)
    .WithReference(ingestionCache)
    .WaitFor(ingestionCache);
```

Reply:
Yeah, the reason was that it makes the template logic a little harder to write (we don't know where the chain ends and therefore don't know where the semicolon should go).

Reply:
ah templates. Yeah, I get it.